processing/src/main/java/org/apache/druid/segment/DimensionHandlerUtils.java

```diff
@@ -60,7 +60,8 @@
                                   .setDictionaryEncoded(false)
                                   .setDictionaryValuesUnique(false)
                                   .setDictionaryValuesSorted(false)
-                                  .setHasBitmapIndexes(false);
+                                  .setHasBitmapIndexes(false)
+                                  .setHasMultipleValues(false);
```

Comment:
The issue with this is that it is used when column capabilities don't exist by `getEffectiveCapabilities`, and `makeVectorProcessor` was not in fact checking if it was `complete`, so the other change in this file to check `isMaybeTrue` causes a ton of test failures because it now it tries to make multi-value dimension processors/selectors for these columns. We could leave it as unknown, but change `makeVectorProcessor` to `isTrue`, but that doesn't seem correct either. Ideally, we remove these default capabilities entirely, but we aren't quite there yet I think.

Reply:
Since null capabilities for vector selectors means unambiguously that the column doesn't exist, I changed this back to leaving multi-value as unknown in favor of checking for this condition in `makeVectorProcessor`, which I think is the best thing to do for now

Reply:
OK, that makes sense.